### PR TITLE
Redirect people to Lightning monorepo in README.

### PR DESCRIPTION
**Pull request by vincentmr**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [ ] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [ ] Ensure that the test suite passes, by running `make test`.

- [ ] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [ ] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**
LK moved to the monorepo.

**Description of the Change:**
Redirect people to monorepo in README.

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

**Comment by vincentmr** [comment]
@AmintorDusko I'm tagging you since you often have nice suggestions concerning documentation. 
@mlxd When we're good, I think you can archive the repo. 
Cheers!

File: README.rst [hunk 1]
```diff
@@ -3,6 +3,9 @@ PennyLane-Lightning-Kokkos Plugin
 
 .. header-start-inclusion-marker-do-not-remove
 
+PennyLane-Lightning-Kokkos was integrated in `PennyLane-Lightning <https://github.com/PennyLaneAI/pennylane-lightning>`_ v0.32, and hence the PennyLane organization stopped maintaining this repository at v0.32.
+The development of PennyLane-Lightning-Kokkos continues under `PennyLane-Lightning <https://github.com/PennyLaneAI/pennylane-lightning>`_ so we encourage you to report `issues <https://github.com/PennyLaneAI/pennylane-lightning/issues>`_, make `pull requests <https://github.com/PennyLaneAI/pennylane-lightning/pulls>`_, etc. there.
+
```
Comment (AmintorDusko) [inline README.rst:8]: Let me know if this sounds good.
```suggestion

Disclosure
========

PennyLane-Lightning-Kokkos was integrated into `PennyLane-Lightning <https://github.com/PennyLaneAI/pennylane-lightning>`. This backend development continues there, and so we direct users to `PennyLane-Lightning <https://github.com/PennyLaneAI/pennylane-lightning>` to report `issues <https://github.com/PennyLaneAI/pennylane-lightning/issues>`_, make `pull requests <https://github.com/PennyLaneAI/pennylane-lightning/pulls>`_, etc.
The PennyLane organization will archive this repository, which will be a Snapshot of v0.32.


```

Comment (vincentmr) [reply README.rst:8]: How about 
```suggestion
Disclosure
==========

The PennyLane organization has archived this repository, which is a snapshot of PennyLane-Lightning-Kokkos v0.32.
This backend was integrated into `PennyLane-Lightning <https://github.com/PennyLaneAI/pennylane-lightning>`_ where its development continues.
We direct users and developers to `PennyLane-Lightning <https://github.com/PennyLaneAI/pennylane-lightning>`_ to report `issues <https://github.com/PennyLaneAI/pennylane-lightning/issues>`_, make `pull requests <https://github.com/PennyLaneAI/pennylane-lightning/pulls>`_, etc.

About
=====
```

File: README.rst [hunk 2]
```diff
@@ -3,6 +3,15 @@ PennyLane-Lightning-Kokkos Plugin
 
 .. header-start-inclusion-marker-do-not-remove
 
+Disclosure
+==========
+
+The PennyLane organization has archived this repository, which is a snapshot of PennyLane-Lightning-Kokkos v0.32.
```
Comment (trbromley) [inline README.rst:9]:
```suggestion
The PennyLaneAI organization has archived this repository, which is a snapshot of PennyLane-Lightning-Kokkos v0.32.
```

File: README.rst [hunk 3]
```diff
@@ -3,6 +3,15 @@ PennyLane-Lightning-Kokkos Plugin
 
 .. header-start-inclusion-marker-do-not-remove
 
+Disclosure
+==========
```
Comment (trbromley) [inline README.rst:7]: I interpreted "disclosure" as a legal term

```suggestion
❗ Warning
=========
```

**Review by mlxd — APPROVED** [review]

**Review by AmintorDusko — APPROVED** [review]

**Pull request merged**